src/communication/network_interface_sr.c

```diff
@@ -11324,3 +11329,36 @@ smmon_get_server_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request,
   css_send_data_to_client (thread_p->conn_entry, rid, reply, OR_ALIGNED_BUF_SIZE (a_reply));
 #endif // !WINDOWS
 }
+
+/*
+ * smmon_finalize_force - finalize memory_monitor forcely
+ *
+ * return:
+ *
+ *  rid(in):
+ *  request(in):
+ *  reqlen(in):
+ */
+void
+smmon_finalize_force (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
+{
+  char *ptr;
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int error = NO_ERROR;
+#if !defined(WINDOWS)
+  mmon_finalize ();
```

Comment:
It would be nice to logging when cub_server receives/handles this request. So, user(DBA) can find/check this log in the error log.